src/pickers/BatchRunWithLocationPicker.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { injectIntl } from "react-intl";
+import { fetchBatchRunWithLocationPicker } from "../actions";
+import { formatMessage, AutoSuggestion, withModulesManager, decodeId } from "@openimis/fe-core";
+import { getBatchRunLabel } from "../utils";
+import _ from "lodash";
+
+
+class BatchRunWithLocationPicker extends Component {
+  
+  state = {
+    batchRuns: [],
+  };
+
+  componentDidMount() {
+    this.setState({ batchRuns: this.props.batchRuns});
+  }
+
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (!_.isEqual(prevProps.batchRuns, this.props.batchRuns)) {
+      this.setState({ batchRuns: this.props.batchRuns });
```

Comment:
Detected by LGTM https://lgtm.com/projects/g/openimis/openimis-fe-claim_batch_js/rev/pr-275dae5699913b7af342472e900af244db36ceb2 - change to `this.setState((_, props) => ({ batchRuns: props.batchRuns }));`
You can read about the issue here: https://lgtm.com/rules/1819283066/